packages/api/src/hooks/useLandingCompany.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import useAuthorize from './useAuthorize';
+import useFetch from '../useFetch';
+
+/** A custom hook that returns the available landing companies of the user's country. */
+const useLandingCompany = () => {
+    const { data: authorize_data } = useAuthorize();
+    const { data, ...rest } = useFetch('landing_company', {
+        payload: { landing_company: authorize_data.country || '' },
```

Comment:
Will update it with the response from get_settings and use `residence || country` instead